src/components/WorkOrder/AppointmentDetails.js

```diff
@@ -16,6 +18,16 @@ import {
 const AppointmentDetails = ({ workOrder, schedulerSessionId }) => {
   const { user } = useContext(UserContext)
 
+  const statusAllowsScheduling = (currentStatus) => {
+    const statuses = [
+      STATUS_CANCELLED.description,
+      STATUS_AUTHORISATION_PENDING_APPROVAL.description,
+      STATUS_COMPLETE.description,
+      STATUS_NO_ACCESS.description,
+    ]
```

Comment:
I like this separate function.

This is a minor suggestion and doesn't block a merge but another way to write this particular part of it would be to map over the statuses and return each description. 

The benefit is that the reader can just read one line to see that we're getting the description for every single status. At the moment someone could add something like `SOME_NEW_STATUS.code` in this array when they mean `.description` and cause a tricky-to-find bug. Using the map approach makes it clearer we're just always interested in the `description` and expresses that in just one place.

E.g.

```
  // ...

  const closedStatusDescriptions = [
    STATUS_CANCELLED,
    STATUS_AUTHORISATION_PENDING_APPROVAL,
    STATUS_COMPLETE,
    STATUS_NO_ACCESS
  ].map((status) => status.description })

  return !closedStatusDescriptions.includes(currentStatus)

  // ...
```

Reply:
I agree with @NeilvB! I am wondering if this best to moved to `src/utils/status-codes.js`, so we have all the related functions in one place.